sql_generators/active_users_aggregates_v3/templates/mobile_query.sql

```diff
@@ -223,19 +207,20 @@ SELECT
     first_seen_date,
     durations
   ),
-  COUNT(DISTINCT IF(days_since_seen = 0, client_id, NULL)) AS daily_users,
-  COUNT(DISTINCT IF(days_since_seen < 7, client_id, NULL)) AS weekly_users,
-  COUNT(DISTINCT client_id) AS monthly_users,
-  COUNT(DISTINCT IF(days_since_seen = 0 AND durations > 0, client_id, NULL)) AS dau,
-  COUNT(DISTINCT IF(submission_date = first_seen_date, client_id, NULL)) AS new_profiles,
+  COUNTIF(is_daily_user) AS daily_users,
```

Comment:
I'm evaluating if it's worth having this in clients_last_seen_joined, since unified_metrics and active_users_aggregates are the only downstreams.